dapps/profile.vue/src/components/did/Delegates.vue

```diff
@@ -80,11 +80,15 @@
           <evan-v-select
             class="m-0"
             :value="null"
-            :options="contacts"
+            :options="filteredContacts"
             :placeholder="'_profile.did.choose-contact' | translate"
             :clear-search-on-select="true"
             @input="onSelectContact"
-          />
+          >
+            <template #no-options>
```

Comment:
What does `#no-options` ?

Reply:
https://vuejs.org/v2/guide/components-slots.html#Named-Slots-Shorthand

Reply:
an ugly but helpful shorthand :D

Reply:
I think we should prefer the shorthand, since we prefer it everywhere else too :)(